Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -174,4 +174,49 @@
   <data name="Global_Continue" xml:space="preserve">
     <value>Continue</value>
   </data>
+  <data name="Header_Service_Name" xml:space="preserve">
```

Comment:
```suggestion
  <data name="Header_ServiceName" xml:space="preserve">
```

Could we tweak the structure of the keys please. I kind of see the underscore as an indicating of increasing specificity within the overall hierarchy of keys. And within each segment of the key CamelCasing is enough to make it readable.

For example, the key `OrganisationRegistration_EnterOrganisationName_Heading`

is in the `OrganisationRegistration` section, on the `EnterOrganisationName` page and it addresses the `Heading`

Reply:
Updated structure of the keys. Think they are ok now.